src/test/scala/no/ndla/imageapi/service/WriteServiceTest.scala

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Image-api har ikke støtte for opplasting av flere bilder samtidig. Fikset doccen.